agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -281,7 +290,7 @@ class EvaluationDB(Document):
     app: Link[AppDB]
     organization: Link[OrganizationDB]
     user: Link[UserDB]
-    status: str = Field(default="EVALUATION_INITIALIZED")
+    status: Result
```

Comment:
I am a bit confused why we are using Result here as a status?

Reply:
so if I understand correctly, we are reusing the Result object for both the EvalScenario and here, but with different meanings. Maybe making another one called EvalFinalStatus or something like this would make the code clearer